k2/python/csrc/torch/fsa_algo.cu

```diff
@@ -686,72 +699,93 @@ static void PybindReplaceFsa(py::module &m) {
 }
 
 static void PybindCtcGraph(py::module &m) {
+  m.def(
+      "ctc_graph",
+      [](RaggedAny &symbols, torch::optional<torch::Device> = {},
```

Comment:
I think we don't need device argument here.

Reply:
The purpose is to make the call `k2.linear_fsa` in Python as uniform as possible: You don't need to know the type of `lables`.
https://github.com/k2-fsa/k2/blob/8988489702372b7e331dcd6815d3e9bb162f4c84/k2/python/k2/fsa_algo.py#L66